test/format/format.i

```diff
@@ -235,6 +252,49 @@ table_select_type(table_type type)
     return (tables[i]);
 }
 
+/*
+ * wiredtiger_open_session --
+ *     Open a WiredTiger session.
+ */
+static inline void
```

Comment:
I'm a bit surprised these are inline - they seem unlikely to be in performance sensitive code.

Reply:
There's no reason these need to be inline. (They were `#define` calls at some point I think, and I converted them so I could set gdb breakpoints.)

Reply:
We will remove the inline keyword.

Reply:
I've created a ticket for all the small changes, including this one: [WT-9563](https://jira.mongodb.org/browse/WT-9563)